tests/tools/src/cuda/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+#Detray library, part of the ACTS project(R& D line)
+#
+#(c) 2024 CERN for the benefit of the ACTS project
+#
+#Mozilla Public License Version 2.0
+
+# C++17 support for CUDA requires CMake 3.18.
+cmake_minimum_required(VERSION 3.18)
+
+# Enable CUDA as a language.
+enable_language(CUDA)
+
+# Set the CUDA build flags.
+include(detray-compiler-options-cuda)
+
+include( CMakeFindDependencyMacro )
+
+find_dependency( Boost COMPONENTS program_options )
+
+# Silence actsvg internal warnings
+include_directories( SYSTEM $<TARGET_PROPERTY:actsvg::meta,INTERFACE_INCLUDE_DIRECTORIES> )
+include_directories( SYSTEM $<TARGET_PROPERTY:actsvg::core,INTERFACE_INCLUDE_DIRECTORIES> )
```

Comment:
Again, please don't use `include_directories(...)`.

Reply:
Without this, I get a bunch of compiler warnings that will kill the CI. How can I supress the warnings from external projects?

Reply:
The best way for doing this would be to massage the `actsvg::meta` and `actsvg::core` libraries to pull in their headers as "system headers". Similar to what we do in:

https://github.com/acts-project/algebra-plugins/blob/main/extern/eigen3/CMakeLists.txt#L39-L44

That's also a bit hacky, but I think that's the next best thing after actually fixing the warnings. :stuck_out_tongue: